### DISCUSSION TITLE
[Question] Pytorch 2.0 with colossalAI

### DISCUSSION THREAD

**Discussion by skyshine102** [hpcaitech/ColossalAI#3150] [Community | Q&A]
Today, pytorch 2.0 has been released and it seems that the backend is quite efficient. I am wondering if colossalai will support pytorch 2.0 in the near future?  
Or someone who has been successfully built the environment with torch 2.0 + colossalai can share experiences?

**Comment by binmakeswell** [comment]
Hi @skyshine102 Thanks for your suggestion! Yes, we are working on it : )

**Reply by skyshine102** [reply]
Looking forward to it! New backend in torch 2.0 can reduce the burden of compiling various fused/ special kernel (i.e. internal flash attention)
